lms/djangoapps/onboarding/choices.py

```diff
@@ -0,0 +1,50 @@
+FUNCTIONS_LABELS = (
+    ('strategy_planning', 'Strategy and planning'),
+    ('leadership_governance', 'Leadership and governance'),
+    ('program_design', 'Program design and development'),
+    ('measurement_eval', 'Measurement evaluation and learning'),
+    ('stakeholder_engagement', 'External relations and partnerships'),
+    ('human_resource', 'Human resource management'),
+    ('financial_management', 'Financial management'),
+    ('fundraising', 'Fundraising and resource mobilization'),
+    ('marketing_communication', 'Marketing communications and PR'),
+    ('system_tools', 'Systems tools and processes'),
+)
+
+INTERESTS_LABELS = (
+    ('strategy_planning', 'Strategy and planning'),
+    ('leadership_governance', 'Leadership and governance'),
+    ('program_design', 'Program design and development'),
+    ('measurement_eval', 'Measurement, evaluation, and learning'),
+    ('stakeholder_engagement', 'External relations and partnerships'),
+    ('human_resource', 'Human resource management'),
+    ('financial_management', 'Financial management'),
+    ('fundraising', 'Fundraising and resource mobilization'),
+    ('marketing_communication', 'Marketing, communications, and PR'),
+    ('system_tools', 'Systems, tools, and processes'),
+)
```

Comment:
Functions and Interests labels are exactly same in previous dictionaries, There was only difference in keys that we changed in current implementation and make them same.  
![Uploading Screen Shot 2020-05-21 at 1.05.26 PM.png…]()

If we are going to keep them same, Then I think we can use single constant for function and interests choices 
`FUNCTIONS_INTERESTS_CHOICES`

Reply:
@saad-waqar @ahmed-faraz46 your thoughts ?

Reply:
For now, I have retained the current implementation keeping in mind that there could be a change in future in one of the set of choices (functions or interests), ensuring separation of concern and loose coupling.

However, if consensus is performed among the reviewers, and we're sure that these fields won't be different in future, I'll update as suggested. After all, we are having 10 duplicate code lines in the current implementation.

Reply:
If they're being used in 2 different fields let's have 2 different dictionaries